tests/test_dimr_bmi.c

```diff
@@ -0,0 +1,22 @@
+#include "dimr_bmi.h"
+#include "unity.h"
+#include <string.h>
+
+void setUp(void) {}
+
+void tearDown(void) {}
+
+static void test_version_number(void) {
```

Comment:
I would call it test_version_string, because the function under test is called get_version_string() and also because you check everything that is *not* a number.

Reply:
Good one, done